### ISSUE TITLE
Notifications should be expanded on hover

### ISSUE THREAD

**Issue by nahuelwexd** [elementary/notifications#66]
Recently #63 has been merged, but this PR implies that all notifications will have 2 lines as maximum, which I think is not always the expected behaviour. For example, sometimes, you want to read the whole msg directly from the notification, without the need of opening the app. So I think it would be cool to add a behaviour which allows to the user to expand notifications when hovering with the cursor

**Comment by cassidyjames** [comment]
I agree expanding on hover could be useful, but I think we should still cap it so it doesn't get out of hand. Maybe up to 2 lines normally, 4 on hover? Notifications should not be used as a way to communicate long-form content, but as a way to notify that something important has happened in an app (and to direct a user to that app).

**Comment by nahuelwexd** [comment]
Yeah, maybe up to a limited amount of lines
